docs/deployment/cli.md

```diff
@@ -335,7 +335,64 @@ parameter combinations:
 The `--target` will overwrite the [execution.target]({% link deployment/config.md %}#execution-target) 
 specified in the `config/flink-config.yaml`.
 
-For more details on the commands and the available options, please refer to the Resource Provider-specific 
+For more details on the commands and the available options, please refer to the Resource Provider-specific
 pages of the documentation.
 
+### Submitting PyFlink Jobs
+
+Currently users are able to submit a PyFlink job via the CLI. It does not require to specify the
+jar file path or the entry main class, which is different from the Java job submission. The followings
+are some example of PyFlink job submission:
+
+<span class="label label-info">Note</span> When submitting Python job via `flink run`, Flink will run the command “python”. Please run the following command to confirm that the command “python” in current environment points to a specified Python version 3.5, 3.6, 3.7 or 3.8:
+{% highlight bash %}
+$ python --version
+# the version printed here must be 3.5, 3.6, 3.7 or 3.8
+{% endhighlight %}
+
+- Run a Python Table job:
+{% highlight bash %}
+$ ./bin/flink run -py examples/python/table/batch/word_count.py
+{% endhighlight %}
+
+- Run a Python Table job with pyFiles:
+{% highlight bash %}
+$ ./bin/flink run -py examples/python/table/batch/word_count.py \
+      -pyfs file:///user.txt,hdfs:///$namenode_address/username.txt
+{% endhighlight %}
+
+- Run a Python Table job with a JAR file:
+{% highlight bash %}
+$ ./bin/flink run -py examples/python/table/batch/word_count.py -j <jarFile>
+{% endhighlight %}
+
+- Run a Python Table job with pyFiles and pyModule:
+{% highlight bash %}
+$ ./bin/flink run -py examples/python/table/batch/word_count.py -j <jarFile>
+{% endhighlight %}
+
+- Run a Python Table job on a specific JobManager:
+{% highlight bash %}
+$ ./bin/flink run -m myJMHost:8081 -py examples/python/table/batch/word_count.py
+{% endhighlight %}
+
+- Run a Python Table job using a [per-job YARN cluster]({% link deployment/resource-providers/yarn.md %}#run-a-single-flink-job-on-hadoop-yarn) with 2 TaskManagers:
+{% highlight bash %}
+$ ./bin/flink run -m yarn-cluster -py examples/python/table/batch/word_count.py
```

Comment:
```suggestion
$ ./bin/flink run --target yarn-per-job -py examples/python/table/batch/word_count.py
```
Isn't `-m` wrong here as it refers to the JobManager?

Reply:
Additionally, we want the format of the command to be aligned with the other commands, i.e. long parameters, 6 space indentation and one parameter per line:
```
./bin/flink run \
      --target yarn-per-job \
      --python examples/python/table/batch/word_count.py
```
You might adapt the parameters to fit to the description accordingly.